packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1072,15 +1155,161 @@ test('lib-wallet addOffer for autoswap swap', async t => {
   );
 });
 
+test('lib-wallet can give attestations in offers', async t => {
+  const {
+    wallet,
+    attestationPublicFacet,
+    attestationBrand,
+    attestationIssuer,
+  } = await setupTest({ attestation: true });
+
+  const issuerManager = wallet.getIssuerManager();
+  await issuerManager.add('Attestation', attestationIssuer);
+  await wallet.makeEmptyPurse('Attestation', 'staking purse');
+
+  const invitation = await attestationPublicFacet.makeReturnAttInvitation();
+
+  const rawId = '1593482020370';
+  const id = `unknown#${rawId}`;
+  const offer = {
+    id: rawId,
+    invitation,
+    proposalTemplate: {
+      give: {
+        Attestation: {
+          pursePetname: 'staking purse',
+          value: 30n,
+          type: 'Attestation',
+        },
+      },
+      exit: {
+        onDemand: null,
+      },
+    },
+  };
+
+  await wallet.addOffer(offer);
+  const accepted = await wallet.acceptOffer(id);
+  assert(accepted);
```

Comment:
Is this supposed to be a test assertion? How about...

```suggestion
  t.true(accepted);
```

Reply:
It's actually to make typescript happy, on the next line it tries to access `{ depositedP } = accepted`, but accepted might be `undefined`.

Reply:
fwiw, that won't be necessary in Ava 4 which adds type guards https://github.com/avajs/ava/releases/tag/v4.0.0

Reply:
ah. type safety. fair enough.